atest/testdata/standard_libraries/string/convert_to.robot

```diff
@@ -19,6 +19,29 @@ Convert To Lowercases
     ${EMPTY}    ${EMPTY}
     ÖÖÄÄÅÅ    ööääåå
 
+Convert To Titlecases
+    [Template]    Do titlecases
+    abcd    Abcd
+    jUnit    jUnit
+    aBBbcd    aBBbcd
+    1234    1234
+    a1b2 C3d4e    A1b2 C3d4e
+    foo\tbar    Foo\tB ar
+    foo 1bar    Foo 1Bar
+    foo's1bar    Foo'S1Bar
+    hello\nXML\niPhone    Hello\nXML\niPhone
+    i love robot Framework    I Love Robot Framework
+    i love ROBOT Framework    I Love ROBOT Framework
+    I Love Robot Framework    I Love Robot Framework 
+    foO baR    foO baR
+    1234    1234
+    mother_in_law    Mother_In_Law
+    father-in-law    Father-In-Law
+    jUnit    jUnit
+    iPhone    iPhone
+    foo\xa0bar    Foo\xa0Bar
+    ${EMPTY}    ${EMPTY}    
```

Comment:
Wow, this is an extensive test! To make it easier to understand, the data could be aligned like this, though:

     abcd                      Abcd
     jUnit                     jUnit
     aBBbcd                    aBBbcd
     i love robot Framework    I Love Robot Framework